requirements_cpu_daal.yml

```diff
@@ -0,0 +1,12 @@
+name: mortgage_cpu_daal
```

Comment:
looks like the file is named wrong, as it's both for "cpu_daal" and "cpu_xgboost" cases

Reply:
fixed